### PR TITLE
Match UITableView's sizing behavior for Swipe Action sizing more closely.

### PR DESCRIPTION
**Pull request by nsillik**
Match `UITableView`'s behavior for sizing the swipe actions.

- Maximum button size is 120pts
- Buttons can span up to two lines, truncating at the end
- Automatically shrink text on too-long text before truncating:

UIKit:
image

These changes:
image

### Checklist

Please do the following before merging:

- [x] Ensure any public-facing changes are reflected in the [changelog](https://github.com/kyleve/Listable/blob/main/CHANGELOG.md). Include them in the `Main` section.

### CODE REVIEW

File: ListableUI/Sources/Internal/SwipeActionsView.swift [hunk 1]
```diff
@@ -182,7 +182,8 @@ final class SwipeActionsView: UIView {
 
         case .sizeThatFits:
             return buttons.map {
-                max($0.sizeThatFits(UIView.layoutFittingCompressedSize).width, style.minWidth)
+                let x = max($0.sizeThatFits(UIView.layoutFittingCompressedSize).width, style.minWidth)
+                return min(120, x)
```
Comment (nsillik) [inline ListableUI/Sources/Internal/SwipeActionsView.swift:186]: extract, possibly make configurable?

File: Demo/Sources/Demos/Demo Screens/SwipeActionsViewController.swift [hunk 2]
```diff
@@ -143,7 +143,7 @@ final class SwipeActionsViewController: UIViewController  {
         return SwipeActionsConfiguration(performsFirstActionWithFullSwipe: true) {
             if allowDeleting {
                 SwipeAction(
-                    title: "Delete",
+                    title: "Delete Delete Delete Delete Delete Delete Delete DeleteDelete Delete Delete Delete Delete Delete Delete DeleteDelete Delete Delete Delete Delete Delete Delete DeleteDelete Delete Delete Delete Delete Delete Delete Delete",
```
Comment (nsillik) [inline Demo/Sources/Demos/Demo Screens/SwipeActionsViewController.swift:146]: undo this change, was just for testing

File: ListableUI/Sources/Internal/SwipeActionsView.swift [hunk 3]
```diff
@@ -265,6 +266,10 @@ private class DefaultSwipeActionButton: UIButton {
 
         titleLabel?.font = .systemFont(ofSize: 15, weight: .medium)
         titleLabel?.lineBreakMode = .byTruncatingTail
+        titleLabel?.numberOfLines = 2
+        titleLabel?.minimumScaleFactor = 0.8
+        titleLabel?.adjustsFontSizeToFitWidth = true
```
Comment (nsillik) [inline ListableUI/Sources/Internal/SwipeActionsView.swift:271]: these are always so ugly, IMO

**Review by kyleve — APPROVED** [review]

**Review by robmaceachern — APPROVED** [review]
Cool cool!
